lib/app/components/atoms/chat_bubble.dart

```diff
@@ -1,6 +1,8 @@
 import 'package:favoritism_communication/app/styles/styles.dart';
-import 'package:chat_bubbles/chat_bubbles.dart';
+import 'package:flutter_chat_bubble/bubble_type.dart';
+import 'package:flutter_chat_bubble/chat_bubble.dart' as cb;
 import 'package:flutter/material.dart';
+import 'package:flutter_chat_bubble/clippers/chat_bubble_clipper_4.dart';
```

Comment:
```suggestion
import 'package:flutter/material.dart';
import 'package:flutter_chat_bubble/chat_bubble.dart';
```
クラス名を変更したら、`flutter_chat_bubble`のパッケージをimportするだけで良さそうです

Reply:
ご指摘の通り修正しました！